.github/workflows/ci.yml

```diff
@@ -638,10 +575,33 @@ jobs:
       - os-family=Linux
     strategy:
       matrix:
-        architecture: [rv64, rv32-linux]
+        target:
+          - platform: android
+            architecture: arm64-v8a
+            abi: arm64-v8a
+            docker_image: "gcr.io/iree-oss/android@sha256:76c2a52dcd6d07601227b965ac87d021c1d2d5e2d01f46ad58da28c89267f2ab"
+            run_scripts: "./build_tools/cmake/build_android.sh"
+          - platform: riscv
+            architecture: rv64
+            abi: lp64d
+            docker_image: "gcr.io/iree-oss/riscv@sha256:d6f0e293a50faf5abbd564c1d1bb9dc6456d7ce93d07b131c381fa64c1daed62"
+            run_scripts: "./build_tools/cmake/build_riscv.sh && ./build_tools/cmake/test_riscv.sh"
```

Comment:
Since the build script always follows `./build_tools/cmake/build_${PLATFORM^^}.sh`, maybe we don't need to put them here but in the step?

This way, we can also separate the build and test step, to make it easier to debug.

Reply:
Done.